apps/voting/app/src/components/VoteDetail/DisputableActionStatus.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import {
+  Box,
+  Button,
+  GU,
+  IconLock,
+  Info,
+  textStyle,
+  Timer,
+  useTheme,
+} from '@aragon/ui'
+import { getAgreement } from '../../agreementsMockData'
+import { addressesEqual } from '../../web3-utils'
+
+function DisputableActionStatus({ vote, connectedAccount }) {
+  //TODO: get agreement and vote real data
+  const theme = useTheme()
+  const agreement = getAgreement()
+
+  return (
+    <React.Fragment>
```

Comment:
Looks like the `<Box>` is acting as the root node so we can make this even tidier by removing this